manual/vcredist2008/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,15 @@
+﻿$packageName = 'vcredist2005'
+$packageSearch = "Microsoft Visual C++ 2008 Redistributable*"
```

Comment:
As I mentioned, it could perhaps help if escaping the `+` characters, or it's possible that it just happens on appveyor if they got several different versions of vcredist 2008 installed.